apis/control/v1alpha1/k8ssandratask_types.go

```diff
@@ -199,3 +175,47 @@ func getConditionStatus(s cassapi.CassandraTaskStatus, conditionType cassapi.Job
 func init() {
 	SchemeBuilder.Register(&K8ssandraTask{}, &K8ssandraTaskList{})
 }
+
+// getCondition converts a primitive boolean into a k8s ConditionStatus
+func getCondition(condition bool) corev1.ConditionStatus {
```

Comment:
Could we name this `toConditionStatus` or something similar? The "get" terminology makes it too close to existing methods like `GetConditionStatus`.